pinot-common/src/main/java/org/apache/pinot/parsers/QueryCompiler.java

```diff
@@ -24,6 +24,10 @@
 /**
  * Interface for Pinot Query compilers.
  */
-public interface AbstractCompiler {
-  BrokerRequest compileToBrokerRequest(String expression);
+public interface QueryCompiler {
```

Comment:
Although I don't think this is a public interface or should create any backward incompatibility (don't expect folks have their own implementations), but still wanted to call this out here.

Reply:
Yeah, this is internal only. Naming an interface `Abstract*` seems very weird to me, so changed it